examples/cartpole/README.md

```diff
@@ -3,32 +3,61 @@
 This example is a recreation of the classic [CartPole](https://www.gymlibrary.ml/environments/classic_control/cart_pole/)
 problem in [Webots](https://cyberbotics.com) as seen in [OpenAI's gym](https://gym.openai.com/).
 
-Two solutions of the problem are provided, one with discrete action space and one with continuous action space.
-The discrete one is solved with the 
-[Proximal Policy Optimization Reinforcement (PPO) Learning (RL) algorithm](https://openai.com/blog/openai-baselines-ppo/).
-The continuous one is solved with the 
-[Deep Deterministic Policy Gradient RL algorithm](https://arxiv.org/abs/1509.02971).
-Both use [PyTorch](https://pytorch.org/) as their backend neural network library.
+Various solutions to the problem are provided, with discrete and continuous action spaces. 
+[PyTorch](https://pytorch.org/) is used as the backend neural network library. 
+Discrete and continuous action space cases are split because switching between them requires several modifications 
+on both the robot and the supervisor controllers.
 
-Switching between discrete and continuous action spaces requires several modifications on both the robot and the 
-supervisor controllers.
+Examples utilize both the [emitter - receiver scheme](https://github.com/aidudezzz/deepbots#emitter---receiver-scheme) and the 
+[robot - supervisor scheme](https://github.com/aidudezzz/deepbots#combined-robot-supervisor-scheme) and act as basic examples of 
+[deepbots framework](https://github.com/aidudezzz/deepbots) in use.
 
-More CartPole examples can be added in the future, using other algorithms/backends.
-Both examples utilize the [emitter - receiver scheme](https://github.com/aidudezzz/deepbots#emitter---receiver-scheme),
-with plans of providing the same examples using the 
-[robot - supervisor scheme](https://github.com/aidudezzz/deepbots#combined-robot-supervisor-scheme) in the future.
+**More CartPole examples can always be added/contributed, using other algorithms/backends.**
 
-## Contents
-- [CartPole discrete](https://github.com/aidudezzz/deepworlds/tree/dev/examples/cartpole/cartpole_discrete)
-- [CartPole continuous](https://github.com/aidudezzz/deepworlds/tree/dev/examples/cartpole/cartpole_continuous)
+----
 
+### Discrete action space
 
-## Solved CartPole - PPO discrete
+The discrete action space case is solved with a custom implementation of the
+[Proximal Policy Optimization Reinforcement (PPO) Learning (RL) algorithm](https://openai.com/blog/openai-baselines-ppo/), 
+both with the [emitter](./cartpole_discrete/controllers/supervisor_manager/supervisor_controller.py)-[receiver](./cartpole_discrete/controllers/robot_controller/robot_controller.py) and 
```

Comment:
These hyperlinks, emitter->supervisor_controller.py and receiver->robot_controller.py, are a bit confusing to me.
They both have an emitter and a receiver.

Reply:
Same as the previous comment 😄